recipes/phconvert/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "phconvert" %}
+{% set version = "0.7" %}
+{% set sha256 = "d4a5335d9f3a5c8df753cb31c6152dbb3d7ded01266a1188804289e0236df2c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/Photon-HDF5/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.9
+    - pytables
+    - future
+
+  run:
+    - python
+    - setuptools
+    - numpy >=1.9
+    - pytables
+    - future
+    
+test:
+  imports:
+    - phconvert
+
+about:
+  home: http://photon-hdf5.github.io/phconvert
+  license: MIT
```

Comment:
I'm using the archive from github which has all the files and I added the license_file line. The next release will also have updated MANIFEST.in but it will not happen soon.